src/main/java/net/javadiscord/javabot/systems/commands/SearchCommand.java

```diff
@@ -0,0 +1,95 @@
+package net.javadiscord.javabot.systems.commands;
+
+import java.net.*;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Path;
+import java.util.*;
+import java.io.*;
+import javax.net.ssl.HttpsURLConnection;
+
+import com.google.gson.*;
+import net.dv8tion.jda.api.EmbedBuilder;
+import net.dv8tion.jda.api.events.interaction.SlashCommandEvent;
+import net.dv8tion.jda.api.requests.restaction.interactions.ReplyAction;
+import net.javadiscord.javabot.Bot;
+import net.javadiscord.javabot.command.Responses;
+import net.javadiscord.javabot.command.SlashCommandHandler;
+import net.javadiscord.javabot.data.config.BotConfig;
+
+public class SearchCommand implements SlashCommandHandler {
+    static BotConfig config = new BotConfig(Path.of("config"));
+    static String subscriptionKey = config.getSystems().azureSubscriptionKey;
+
+    static String host = "https://api.bing.microsoft.com";
+    static String path = "/v7.0/search";
```

Comment:
This stuff doesn't need to be static. Convert the `subscriptionKey` to a local variable and make the other stuff private and final. And since the host- & path strings are constants, change their name to `HOST` & `PATH` to match Javas naming conventions